Adirontech/trek-webapp
# Land Usage Planner Navbar Full Stack

# Testing Instructions
1. Spin up the project locally (db, backend, and frontend).
2. From the browser, create a new account (http://localhost:3000/login). At the end of the create account form, select the checkbox to indicate that you are a land usage planner. 
![Screenshot 2024-03-04 at 10 08 40 PM](https://github.com/Adirontech/trek-webapp/assets/54957497/7e667657-d63e-4ba0-891c-92845e27c6fd)
3. After clicking the "Create Account" button, you should be taken back to the home page. Ensure that there is a link called "Land Usage Planning" in the navbar. 
![Screenshot 2024-03-04 at 10 12 33 PM](https://github.com/Adirontech/trek-webapp/assets/54957497/2f0725c5-319e-46c8-894f-2e30584e8f66)
4. Check PGAdmin to make sure the user was created in the ranger table.
5. Repeat this process, but do not select the checkbox and ensure that you do not see the "Land Usage Planning" link in the navbar.


# Known Limitations
This only supports when users create an account. This is not supported when users login to an account.
